lib/Agrammon/Model.pm6

```diff
@@ -516,4 +535,45 @@ class Agrammon::Model {
         };
         %$!distribution-map-cache
     }
+
+    #| Get a CSV formatted template of all model inputs
+    #| with one demo instance for all multi-instance modules.
+    method get-input-template($sort) {
```

Comment:
I don't really think CSV formatting belongs in `Agrammon::Model`; perhaps move this logic out to the service class?